third_party/zlib/arm_features.c

```diff
@@ -86,8 +86,18 @@ static void _arm_check_features(void)
       SbLogFlush();
     }
 #elif defined(ARMV8_OS_ANDROID) && defined(__aarch64__)
```

Comment:
Can we change this line to:
 #elif defined(ARMV8_OS_ANDROID) && defined(__aarch64__) && !defined(NATIVE_TARGET_BUILD)

Reply:
I moved `&& !defined(NATIVE_TARGET_BUILD)` up to that line and removed the nested conditional. I had to also add `&& !defined(NATIVE_TARGET_BUILD)` to the next `#elif`, though, to keep it from catching this native target build for arm64.

Why do you want to change `__aarch64__` to `aarch64`?  `__aarch64__` should be set by the toolchain and I'd rather not modify this third-party code.

Reply:
Lgtm. The automatic Markdown changed `__aarch64__` to be __bold__ __aarch64__, not my intention, haha.